src/CSSBattle/11_EyeOfSauron/Yongbin.html

```diff
@@ -0,0 +1,58 @@
+<div class="center">
+  <div class="smallcircle"></div>
+</div>
+<div class="semicircle1"></div>
+<div class="semicircle2"></div>
+<style>
+  .center {
+    width: 100px;
+    height: 100px;
+    position: fixed;
+    background: #191210;
+    border-radius: 50%;
+    border: solid #eca03d 2px;
+    border-width: 20px;
+  }
+  .smallcircle {
+    width: 50px;
+    height: 50px;
+    background: #84271c;
+    position: fixed;
+    border-radius: 50%;
+    top: 125;
+    left: 175;
```

Comment:
Consider adding `px` even though the code works; minor change to improve readability

Other occurrences found in file as well!
```suggestion
    top: 125px;
    left: 175px;
```

Reply:
Resolved in commit 5ec733b.